api/api.proto

```diff
@@ -179,6 +179,18 @@ enum OperationFlags {
   OP_FLAGS_DETACH_FORCE = 2;
 }
 
+enum VolumePlacementEnforcementType {
+  Required = 0;
+  Preferred = 1;
+}
+
+enum LabelSelectorOperator {
```

Comment:
Move this to `LabelSelectorRequirement` to have the scope and namespace automatically setup.

Reply:
Done